spec/message.abnf

```diff
@@ -36,11 +36,11 @@ quoted-char = %x0-5B         ; omit \
             / %x7D-D7FF      ; omit surrogates
             / %xE000-10FFFF
 
-; based on https://www.w3.org/TR/xml/#NT-Nmtoken,
-; but cannot start with U+002D HYPHEN-MINUS or U+003A COLON ":"
+; similar to https://www.w3.org/TR/xml/#NT-Nmtoken
```

Comment:
I think this is distracting. The similarity to (and differences from) Nmtoken is noted in the spec. It doesn't really add anything to the understanding of the ABNF. I think I'd rather comment on what `unquoted` is:

```suggestion
; literals that do not require quoting: 
; containing no whitespace and with a restricted range of characters
```